google-cloud-tools-plugin/src/com/google/gct/idea/appengine/validation/ApiNamespaceInspection.java

```diff
@@ -75,7 +75,12 @@ public void visitAnnotation(PsiAnnotation annotation) {
           return;
         }
 
-        if(!annotation.getQualifiedName().equals(GctConstants.APP_ENGINE_ANNOTATION_API_NAMESPACE)) {
+        String annotationQualifiedName = annotation.getQualifiedName();
+        if (annotationQualifiedName == null) {
```

Comment:
I think you suggestion is by far a better approach. I simply switched the order of comparison for all the cases.